app/javascript/src/members/activities/activity.js

```diff
@@ -184,12 +197,29 @@ Object.defineProperties(Activity.prototype, {
      */
     value: function (method) {
       var activity = this;
+      var par_notes;
+      var notes = [];
+      console.log(activity.has_notes_checkboxes);
+      if (activity.has_notes_checkboxes()) {
```

Comment:
I also know little JS, however, does it not have switch cases?

Reply:
Is there any way to define the enum in 1 place like in `activity.rb` and reference it from the jabbascript?

Reply:
Don't think so, atleast not trivally